test/browser/test_sdl2_key.c

```diff
@@ -62,13 +62,13 @@ int main(int argc, char **argv) {
 
   SDL_StartTextInput();
 
-  emscripten_run_script("keydown(38);keyup(38)"); // up
-  emscripten_run_script("keydown(40);keyup(40);"); // down
-  emscripten_run_script("keydown(37);keyup(37);"); // left
-  emscripten_run_script("keydown(39);keyup(39);"); // right
-  emscripten_run_script("keydown(65);keyup(65);"); // a
-  emscripten_run_script("keydown(66);keyup(66);"); // b
-  emscripten_run_script("keydown(100);keyup(100);"); // trigger the end
+  emscripten_run_script("keydown(38,  'ArrowUp');   keyup(38,  'ArrowUp')"); // up
+  emscripten_run_script("keydown(40,  'ArrowDown'); keyup(40,  'ArrowWDown')"); // down
```

Comment:
```suggestion
  emscripten_run_script("keydown(40,  'ArrowDown'); keyup(40,  'ArrowDown')"); // down
```

Reply:
Is this string not used, if it did not cause an error when it was wrong?

Reply:
It seems for this particular key press, this test is only checking for the `SDL_KEYDOWN` event.    We could remove the `keyup` of the arrow keys without breaking the test.   Not sure it worth doing that though.

Reply:
Fixed.